datahub-web-react/src/app/entity/dataset/DatasetEntity.tsx

```diff
@@ -256,6 +256,7 @@ export class DatasetEntity implements Entity<Dataset> {
         return {
             urn: entity?.urn,
             name: entity.properties?.name || entity.name,
+            expandedName: entity.name,
```

Comment:
We have a new field called entity.properties.fullyQualifiedName which we should really use here - this name field should be considered opaque going forward